demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -168,7 +169,7 @@ def teardown_class(cls):
         os.remove(CONF_PATH)
         for dir_to_delete in cls.CREATED_DIRS:
             if os.path.exists(dir_to_delete):
-                os.rmdir(dir_to_delete)
+                shutil.rmtree(dir_to_delete)
```

Comment:
I think this change is redundant (probably others), please go over the PR and revert unnecessary changes